core-java-modules/core-java-arrays-guides-2/src/test/java/com/baeldung/arrays/NullArrayTest.java

```diff
@@ -0,0 +1,39 @@
+package com.baeldung.arrays;
+
+import org.assertj.core.api.Assertions;
+import org.junit.Test;
```

Comment:
This import belongs to JUnit 4, let's use JUnit 5 instead.

Reply:
Done